snooty/n.py

```diff
@@ -356,7 +356,7 @@ class Directive(Parent[Node]):
     domain: str
     name: str
     argument: MutableSequence["Text"]
-    options: Dict[str, str]
+    options: Dict[str, Any]
```

Comment:
Can this be `object` instead of `Any` without causing things to blow up?

Reply:
the mentioned blow up does happen. i can keep this to string and parse into a float on the front end

Reply:
I actually don't have a strong opinion here. Moderate preference for sticking with a float until the frontend, I think? But your call!

Reply:
no strong preference! making it flexible with [str, Union[str, float]] was not pretty either

Reply:
discussed offline. making this `Any` or `Union[str, float]` still produces AST output of string. will keep as a string option